pg_search/src/index/directory/mvcc.rs

```diff
@@ -96,29 +97,46 @@ pub struct MVCCDirectory {
     pin_cushion: Arc<Mutex<Option<PinCushion>>>,
 }
 
+impl Debug for MVCCDirectory {
```

Comment:
It looks like maybe this custom implementation isn't necessary anymore, because `PgSearchRelation` now implements `Debug`?